client/app/components/OAuthButton/index.jsx

```diff
@@ -0,0 +1,45 @@
+// @flow
+import React from 'react';
+import { I18n } from 'libs/i18n';
+import css from './OAuthButton.scss';
+import googleIcon from './googleIcon.svg';
+import facebookIcon from './facebookIcon.svg';
+
+type Props = {
+  signIn?: boolean,
+  type: 'facebook' | 'google',
+  action: string,
+  token: string,
+};
+
+export const OAuthButton = ({
+  signIn, type, action, token,
+}: Props) => {
+  const value = I18n.t(`devise.shared.sign_${signIn ? 'in' : 'up'}_${type}`);
```

Comment:
Nitpick: Do you think that a more meaningful variable name would be useful here? I see that `value` is tied to the `button` attribute, but maybe like `oAuthButtonText` (not the best... 😄 )